gems/manageiq_ansible_tower/manageiq_ansible_tower.gemspec

```diff
@@ -0,0 +1,36 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'manageiq_ansible_tower/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = "manageiq_ansible_tower"
+  spec.version       = ManageiqAnsibleTower::VERSION
+  spec.authors       = ["Brandon Dunne"]
+  spec.email         = ["bdunne@redhat.com"]
+
+  spec.summary       = %q{Ansible Tower Provider for ManageIQ}
+  spec.description   = %q{Ansible Tower Provider for ManageIQ}
```

Comment:
Yeah, I was wondering whether this should actually be a separate gem at all. It is indeed not the real provider, so seems a pointless extra abstraction to me.

(My expectation is to never release `manageiq_foreman` as a gem, and instead to eventually fold it into `manageiq-providers-foreman`, either as a separate set of "implementation detail" classes, or by directly merging them with their Provider equivalents.)